packages/zoe/src/types.js

```diff
@@ -349,17 +273,82 @@
  * @property {Issuer} issuer
  * @property {AmountMath} amountMath
  *
- * @typedef {Object} InstallationRecord
- * @property {InstallationHandle} handle - opaque identifier for the installation
- * @property {SourceBundle} bundle - contract code
- *
- * @typedef {Object} OfferStatus
- * @property {OfferHandle[]} active
- * @property {OfferHandle[]} inactive
- *
- * @typedef {(offerHandle: OfferHandle) => void} OfferHook
+ * @typedef {Record<Keyword,Amount>} Allocation
+ * @typedef {Record<Keyword,AmountMath>} AmountMathKeywordRecord
+ */
+
+/**
+ * @typedef {Object} ZCFSeat
+ * @property {() => void} exit
+ * @property {(msg: string=) => never} kickOut
+ * @property {() => Notifier<Allocation>} getNotifier
+ * @property {() => boolean} hasExited
+ * @property {() => ProposalRecord} getProposal
+ * @property {(keyword: Keyword, brand: Brand) => Amount} getAmountAllocated
+ * The brand is used for filling in an empty amount if the `keyword`
+ * is not present in the allocation
+ * @property {() => Allocation} getCurrentAllocation
+ * @property {(newAllocation: Allocation) => Boolean} isOfferSafe
+ * @property {(newAllocation: Allocation) => SeatStaging} stage
+ */
+
+/**
+ * @typedef {Object} SeatStaging
+ * @property {() => ZCFSeat} getSeat
+ * @property {() => Allocation} getStagedAllocation
+ */
+
+/**
+ * @callback OfferHandler
+ * @param {ZCFSeat} seat
+ * @returns any
+ */
+
+/**
+ * @typedef {Object} ContractStartFnResult
+ * @property {Object=} creatorFacet
+ * @property {Promise<Invitation>=} creatorInvitation
+ * @property {Object=} publicFacet
+ */
+
+/**
+ * @callback ContractStartFn
+ * @param {ContractFacet} zcf
+ * @param {Object} terms
+ * @returns {ContractStartFnResult}
+ */
+
+/**
+ * @typedef {Payment<'ZoeInvitation'>} Invitation
+ */
+
+/**
+ * @typedef {{}} Instance
+ */
+
+/**
+ * @template T
+ * @callback GetAmountMath
+ * @param {Brand<T>} brand
+ * @returns {AmountMath<T>}
+ */
```

Comment:
No, it is used.

Reply:
I see. In zoe.js. That makes sense. In that case, could you change the declaration above of
```
 * @property {(brand: Brand) => AmountMath } getAmountMath
```
to
```
 * @property {GetAmountMath} getAmountMath
```
since that seems to be our other general convention?

Reply:
Yup!